JsonParser.h

```diff
@@ -1,13 +1,54 @@
+/**
+ * \class JsonParser
+ *
+ * \brief JsonParser class
+ *
+ * This class declares the JsonParser object, which is used to correctly read the heroes attributes.
+ *
+ * \author NullPointerException
+ *
+ * \version 1.0
+ *
+ * \date 2020/11/01 11:00
+ *
+ */
+
 #ifndef JsonParser_h
 #define JsonParser_h
 #include <map>
 
+//This is the class of the JsonParser
 class JsonParser{
 private:
+    
+    /**
+     * \brief Json file format checking, and cleaning out the blank spaces
+     *
+     * \return Returns a string that has the json files data
+     */
     static std::string WhitespaceCleanerAndFormatChecker(std::string&);
+
 public:
+
+    /**
+     * \brief This method is used to find the hearo's attributes. It looks for the hero's name, health, damage and attackcooldown and maps them together.
+     *
+     * \return Returns the correct Map file of a hero if everythings good. If there's a problem we get a "Bad mapping" warning.
```

Comment:
1. "hearo's" (elírás)
2. itt (és másik JsonParser-beli függvényben is) nem warning van